Src/EB/AMReX_EB2_Level.H

```diff
@@ -476,6 +490,12 @@ GShopLevel<G>::define_fine (G const& gshop, const Geometry& geom,
                 nsmallcells += nsm;
                 nmulticuts  += nmc;
             }
+            // EY: Here -> make plot file
+            if (plt_multiple_cuts)
```

Comment:
Move after the ParallelAllReduce for `nmulticuts`, then add `nmulticuts > 0 && !cover_multiple_cuts` to the conditional